applications/matter_bridge/Kconfig.sysbuild

```diff
@@ -8,15 +8,42 @@
 config NRF_DEFAULT_IPC_RADIO
 	default y
 
+# Use HCI IPC as radio core image for nRF54H20 + nRF7002EK
+config NRF_DEFAULT_BLUETOOTH
+	default y if SOC_SERIES_NRF54HX && WIFI_NRF700X
+
+# Enable IEEE802.15.4 serialization to network core
+config NETCORE_IPC_RADIO_IEEE802154
+	default y if (SOC_SERIES_NRF53X || SOC_SERIES_NRF54HX) && !WIFI_NRF700X
+
+
 # Enable Bluetooth serialization to network core
 config NETCORE_IPC_RADIO_BT_HCI_IPC
-	default y
+	default y if (SOC_SERIES_NRF53X || SOC_SERIES_NRF54HX)
 
 #### Bootloader
 choice BOOTLOADER
-	default BOOTLOADER_MCUBOOT
+	default BOOTLOADER_MCUBOOT if !BOARD_NRF21540DK && !SOC_SERIES_NRF54HX
```

Comment:
We can remove condition for nRF21540, as it's not supported in this application.